src/test/java/com/artipie/rpm/meta/XmlOthersTest.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2019 Yegor Bugayenko
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package com.artipie.rpm.meta;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.core.IsEqual;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+
+/**
+ * Testcase for {@link XmlOthers} class.
+ * @since 0.1
+ */
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class XmlOthersTest {
+
+    /**
+     * Path of the file.
+     */
+    private static String path = "src/test/resources/other.xml";
+
+    /**
+     * The XmlOthers object.
+     */
+    private static XmlOthers xmlOthers = new XmlOthers(Path.of(XmlOthersTest.path));
+
+    /**
+     * File object for the created file.
+     */
+    private static File file = new File(XmlOthersTest.path);
+
+    /**
+     * Expected content of the file.
+     * @checkstyle StringLiteralsConcatenationCheck (20 lines)
+     */
+    private static String expectedOut =
```

Comment:
@ammaratef45 please don't use hard-coded XML expected strings. You can use XML matchers instead to verify XML values by XPATH: https://github.com/jcabi/jcabi-matchers

For example: https://github.com/artipie/rpm-adapter/commit/493153a41dfc9dcae2bb43d8af290adf7d7acf47#diff-f09665f10542cce1ccc41be8358020d1L49-L52